pkg/mimir/modules.go

```diff
@@ -825,6 +831,7 @@ func (t *Mimir) setupModuleManager() error {
 	mm.RegisterModule(ActiveGroupsCleanupService, t.initActiveGroupsCleanupService, modules.UserInvisibleModule)
 	mm.RegisterModule(Distributor, t.initDistributor)
 	mm.RegisterModule(DistributorService, t.initDistributorService, modules.UserInvisibleModule)
+	mm.RegisterModule(EphemeralChecker, t.initEphemeralChecker)
```

Comment:
Let's mark it as "user invisible" module.

Reply:
makes sense, done: 5c25bbf35e22794aefbd58bd08a8fa49d5adf07b